manager/usermanager/createUser.go

```diff
@@ -38,8 +35,26 @@ func CreateUser(userStore *store.UserStore, user *models.UserCredentials) (*mode
 			UnverifiedEmail: user.UserName,
 			Kind:            models.LocalAuth,
 			Role:            models.RoleUser,
+			State:           models.StateCreated,
+			OnBoardingState: models.BoardingStateSignup,
+		}
+	} else {
+		newUser = &models.UserCredentials{
```

Comment:
What is this else case for?
User created by admin?

Reply:
If yes, should we set this to UnverifiedComplete as the state?

Reply:
Yes its for the user created by admin

Reply:
Okay yes It should be UnverifiedComplete. Will make the change